lib/rules/web/ocm_console/cluster_detail_access_control_tab.xyaml

```diff
@@ -162,4 +162,110 @@ click_add_user_button_on_page:
 click_cancel_add_user_button:
   element:
     <<: *cancel_add_user_button
+    op: click
+check_basic_elements_on_AWS_infrastructure_access_tab:
+  elements:
+  - selector:
+      xpath: //div/h3[text()='AWS infrastructure access']
+    timeout: 300
```

Comment:
Just a question, do we need so long time(300s) wait for a dialog? A reasonable timeout(<=30) at here should be better, if we can not get the result in a long time, it is a performance issue.